integration-test/tests/email-autofill.extension.spec.js

```diff
@@ -0,0 +1,57 @@
+import {setupServer, withChromeExtensionContext} from '../helpers/harness.js'
+import { test as base, expect } from '@playwright/test'
+
+/**
+ *  Tests for email autofill in chrome extension.
+ *
+ *  Special setup is needed to load the extension, see withChromeExtensionContext();
+ */
+const test = withChromeExtensionContext(base)
+
+test.describe('Ensure email autofill works in chrome extension', () => {
+    let server
+    test.beforeAll(async () => {
+        server = setupServer()
+    })
+    test.afterAll(async () => {
+        server.close()
+    })
+    test('should select and populate email autofill', async ({page}) => {
+        const selector = '[data-ddg-inputtype="identities.emailAddress"]'
+        const email = 'shane-123@duck.com'
```

Comment:
Can we use global constants for these? I noticed that this is already different to what's in the `apple.spec` file and it's bound to create confusion.